data/themes/darktable.css

```diff
@@ -855,6 +855,12 @@ dialog .dialog-vbox notebook stack box box checkbutton label  /* fix a margin on
   color: @bauhaus_fg;
 }
 
+#bauhaus-combobox *,
+#bauhaus-slider *
```

Comment:
No need to set a specific background here. To be sure background it's the same as other popover menus, add those 2 bauhaus lines around line 900. Around line 900, that will give you that:

```
context-menu,
menu,
menuitem > menu, /* sub-menu */
menuitem arrow,
tooltip,
popover,
combobox window,
dialog combobox window,
#bauhaus-combobox *,
#bauhaus-slider *,
#modulegroups-iop-popup
{
  opacity: 1;  /* this is needed for tooltip on/off feature with Shift+t shortcut */
  background-color: @tooltip_bg_color;
  border-radius: 2px;
  border: 1px solid @button_border;
  outline-style: none;
  min-height: 0.4em;
  min-width: 0.4em;
  padding: 0.2em 0;
}
```
Settings allow to have consistency for all that items. Tested on my side and all good